web/client/src/App.js

```diff
@@ -15,22 +15,37 @@ import "./styles/app.css";
  * @return {MaterialUI} themed app
  */
 function App() {
-  const [token, setToken] = useState("");
-  const [responseVisable, setResponseVisable] = useState(false);
-
+  // const defaultValues = {
+  //   tokenType: "",
+  //   tokenFormat: "",
+  //   tokenScopes: [],
+  //   tokenAudience: [],
+  //   tokenCredentials: "",
+  //   saveTokenLocally: false,
+  // };
+  // const [values, setValues] = useState(defaultValues);
```

Comment:
Don't forget to remove these comments if you don't need them anymore

Reply:
Removed!